src/Lykke.Service.PayInternal.Services/EthereumIataApiClient.cs

```diff
@@ -133,8 +134,7 @@ public class EthereumIataApiClient : IBlockchainApiClient
 
             if (response is ApiException ex)
             {
-                _log.WriteWarning(nameof(CreateAddressAsync), "New erc20 address generation",
-                    ex.Error?.Message);
+                _log.Warning("New erc20 address generation", context: ex);
```

Comment:
Why exception is put in context insead of exception parameter?

Reply:
It is not an exception. ApiException is a plain DTO class declared in EthereumCore client library.